format/src/main/java/com/github/sadikovi/riff/Statistics.java

```diff
@@ -472,4 +477,107 @@ public String toString() {
       return "UTF8[hasNulls=" + hasNulls() + ", min=" + min + ", max=" + max + "]";
     }
   }
+
+  /**
+   * Boolean values statistics.
+   */
+  static class BooleanStatistics extends Statistics {
+    public static final byte ID = 16;
+
+    // whether or not statistics have min/max set
+    protected boolean hasValues = false;
+    // associated with min value
+    protected boolean min = true;
```

Comment:
Decided not to do it, we have exception thrown if method is called on unset statistics.